datastore/tests/unit/test_entity.py

```diff
@@ -221,6 +222,18 @@ def test___repr___w_key_non_empty(self):
         expected = "<Entity%s %s>" % (flat_path, entity_vals)
         self.assertEqual(repr(entity), expected)
 
+    def test__set_item_enum(self):
+        """Test if given Enum converts to int."""
+        entity = self._make_one(key=_Key())
+        entity["some_value"] = _IntEnum.VALUE_1
+
+        self.assertEqual(type(entity["some_value"]), int)
+        self.assertEqual(entity["some_value"], _IntEnum.VALUE_1.value)
```

Comment:
This test doesn't currently reflect how I expect this to be used. I Expect most users will be doing

```suggestion
        self.assertEqual(entity["some_value"], _IntEnum.VALUE_1)
        self.assertEqual(entity["some_value"], _IntEnum.VALUE_1.value)
```

Can this test be changed to assert the pattern we expect to be used?

Reply:
Sure, I've tried it minute ago, test was success